techniques/general/G9.html

```diff
@@ -2,121 +2,102 @@
-            content that meets WCAG 2.1.
+         <h2>達成方法に関する重要な情報</h2>
+         <p>これらの達成方法 (参考) の使用法及び、それらが WCAG 2.1 達成基準 (規定) とどのように関係するかに関する重要な情報については、<a href="https://w3c.github.io/wcag/understanding/understanding-techniques">WCAG 達成基準の達成方法を理解する</a>を参照のこと。適用 (対象) セクションは、その達成方法の範囲について説明しており、特定の技術に関する達成方法があるからといって、WCAG 2.1 を満たすコンテンツを作成する際に、常にその技術が使用可能であるわけではない。
          </p>
       </section>
       <main>
          <section id="applicability">
-            <h2>Applicability</h2>
-            <p>Applies to all technologies that present audio visual information.</p>
-            <p>This technique relates to:</p>
+            <h2>適用 (対象)</h2>
+            <p>視聴覚の情報を提示する全てのウェブコンテンツ技術</p>
+            <p>これは、次の達成基準に関連する達成方法である:</p>
             <ul>
-               <li><a href="https://w3c.github.io/wcag/understanding/captions-live">Success Criterion 1.2.4: Captions (Live)</a> (Sufficient, together with <a href="../general/G93">G93: Providing open (always visible) captions</a>)
+               <li><a href="https://w3c.github.io/wcag/understanding/captions-live">達成基準 1.2.4 (キャプション (ライブ) )</a> (十分、<a href="G93">G93: オープン (常に見える) キャプションを提供する</a> とともに)
                </li>
-               <li><a href="https://w3c.github.io/wcag/understanding/captions-live">Success Criterion 1.2.4: Captions (Live)</a> (Sufficient, together with <a href="../general/G87">G87: Providing closed captions</a>)
+               <li><a href="https://w3c.github.io/wcag/understanding/captions-live">達成基準 1.2.4 (キャプション (ライブ) )</a> (十分、<a href="G87">G87: クローズドキャプションを提供する</a> とともに)
                </li>
-               <li><a href="https://w3c.github.io/wcag/understanding/captions-live">Success Criterion 1.2.4: Captions (Live)</a> (Sufficient, together with <a href="../general/G87">G87: Providing closed captions</a> using a more specific technique)
+               <li><a href="https://w3c.github.io/wcag/understanding/captions-live">達成基準 1.2.4 (キャプション (ライブ) )</a> (十分、より具体的な手法を使用した <a href="G87">G87: クローズドキャプションを提供する</a> とともに)
                </li>
             </ul>
          </section>
          <section id="description">
-            <h2>Description</h2>
-            <p>The objective of this technique is to allow users who cannot hear to be able
-               to access real-time synchronized media broadcasts. It is more difficult to create
-               accurate real-time captions because there is little time to correct mistakes
-               or to listen a second time or consult someone to be sure the words are
-               accurately reproduced. It is also harder to simplify or paraphrase
-               information if it is flowing too quickly.
+            <h2>解説</h2>
+            <p>この達成方法の目的は、音声を聞くことができない利用者がリアルタイムに同期したメディア放送にアクセスできるようにすることである。間違いを訂正するか、二度聞き又は単語が正確に再現されていることを確認するため誰かに意見を聞く時間がほとんどないので、正確なリアルタイムのキャプションを作成することはより困難である。情報があまりに速く流れている時は、簡略化又は言い換えもさらに困難である。
```

Comment:
間違いを訂正するか、二度聞き又は単語が正確に再現されていることを確認するため誰かに意見を聞く時間がほとんどないので、
↓
間違いを訂正する又は、二度聞きもしくは単語が正確に再現されていることを確認するために誰かの意見を聞く時間はほとんどないので、

"or"を明確化してみました。

原文：
> It is more difficult to create accurate real-time captions because there is little time to correct mistakes or to listen a second time or consult someone to be sure the words are accurately reproduced.